vendor/src/github.com/bsphere/le_go/certs.go

```diff
@@ -0,0 +1,222 @@
+package le_go
+
+// Certs from https://logentries.com/doc/certificates
+var pemCerts = []byte(`
```

Comment:
@cpuguy83 @endophage @NathanMcCauley using `SystemCertPool()` works, but will require Go 1.7.

Seems like docker is already being compiled under Go 1.7, so, I guess this is alright...